design/move-gh-org.md

```diff
@@ -0,0 +1,89 @@
+- [ ] Update Netlify deploy settings. Any of the new repo owners; TBD
+- [ ] GH app: Netlify integration. Who: Any of the new repo owners; TBD
+- [ ] GH app: Slack integration. Who: Any of the new repo owners; TBD
+- [ ] Add webhook: travis CI. Who: Any of the new repo owners; TBD
+- [ ] Add webhook: zenhub. Who: Any of the new repo owners; TBD
+- [ ] Move all 3 native provider plugins into their own individual repo. https://github.com/heptio/velero/issues/1537. Who: @carlisia.
+- [ ] Merge PRs from the "pre move" section
+
+### Notes/How-Tos
+
+#### Transfering the GH repository
+
+All action items needed for the repo transfer are listed in the Todo list above. For deailts about what gets moved and other info, this is the GH documentation: https://help.github.com/en/articles/transferring-a-repository
+
+[Pending] We will find out this week who will be the organization owner(s) who will accept this transfer in the new GH org. This organization owner will make all current owners in this repo owners in the new org Velero repo.
+
+#### Updating Travis CI
+
+Someone with owner permission on the new repository needs to go to their Travis CI account and authorize Travis CI on the repo. Here are instructions: https://docs.travis-ci.com/user/tutorial/.
+
+After this, a webhook notifications can be added following these instructions: https://docs.travis-ci.com/user/notifications/#configuring-webhook-notifications.
+
+#### Transfering ZenHub
+
+Pre-requisite: A new Zenhub account must exist for a vmware or vmware-tanzu organization.
+
+This page contains a pre-miration checklist for ensuring a repo migration goes well with Zenhub: https://help.zenhub.com/support/solutions/articles/43000010366-moving-a-repo-cross-organization-or-to-a-new-organization. After this, a webhook will need to be added for desired notifications; instructions here: https://docs.travis-ci.com/user/tutorial/.
+
+After this, webhooks can be added by following these instructions: https://github.com/ZenHubIO/API#webhooks.
+
+#### Updating Netlify
+
+The settings for Netflify should remain the same, except that it now needs to be installed in the new repo. The instructions on how to install Netlify on the new repo are here: https://www.netlify.com/docs/github-permissions/.
+
+#### Communication strategy
+
+[Pending] We will find out this week how this move will be communicated to the community. In particular, the Velero repository move might be tied to the move of our provider plugins into their own repos, also in the new org: https://github.com/heptio/velero/issues/1814.
+
+#### TBD
+
+Many items on the todo list must be done by a repository member with owner permission. This don't all need to be done by the same person obviously, but we should specify if @skriss wants to split these tasks with any other owner(s).
+
+#### Other notes
+
+Might want to exclude updating documentation prior to v0.10.
+GH documentation does not specify if branches on the server are also moved.
```

Comment:
My assumption would be that they would transfer, that said are we using any branches except for master? Looks like we have some old `release-` branches, a `plugin` branch and one for #1821 which I'll delete when that gets merged. Might be a good time to clean up stale branches?

Reply:
I would assume that as well, but did not find it specified. I think the release branches are useful to keep around. Others can be removed. Will enquire if everyone wants to discard the release branches.

@heptio/ark-team?

Reply:
I think some of the releases tags were made from the release branches, mostly point releases. I would hope we keep at least those.